### ISSUE TITLE
photo : get metadata from OM

### ISSUE THREAD

**Issue by JLZIMMERMANN** [organicmaps/organicmaps#3171]
Thanks for your great job.
In my idea there is still a big wish in OpenStreetMap comunity : give ability to illustrate POI.
A good starting point would be to let take pictures with OSM tags in metadata.

Afterward, we would create specific services to store the pictures creating queries the same way as OSm objects.

**Comment by janusbenissa** [comment]
So if I understand correctly you want OM to give the possibility to take pictures of POI?
This while OSM does not support that possibility and there is also nothing provided to be able to save those photos somewhere?

**Comment by JLZIMMERMANN** [comment]
Yes : I wish a way to give a visual description of any object that is already in OpenStreetMap database.

We need to let such ability grow, instead another good option may be offered.
At the moment we already can use :
- Flickr an push the URL on OSM objetc
- commons (wikimedia foundation) an link with the URL of the picture
- Mapillary to create sequence and link with the ID of the picture or sequence
- the webstite of the dev of StreetComplete that let take picture while creating notes example [here](https://www.openstreetmap.org/note/3311978#map=19/44.13137/4.81308&layers=N)

**Comment by biodranik** [comment]
Sorry, I still don't get your idea. What exactly do you want and for which use case? Can you describe it as a real-life example with all details?

**Comment by endim8** [comment]
Uploading & displaying from Commons would be the best imo, especially consoderting there's already a `wikidata_commons=*` tag commonly used in OSM.
The main concern other than actually implementing it would be the storage usage. Perhaps only thumbnails could be stored offline?

**Comment by JLZIMMERMANN** [comment]
`The main concern other than actually implementing it would be the storage usage.`

About the storage I use **Flickr** and I already have about 50,000 photo in this acount. In the other side I also use **Mapillary**.
I think it would be good to let use various option.

**Comment by richlv** [comment]
Is this issue about taking photos and uploading them, or displaying them in OM?
My suggestion is perhaps smaller in scope, and it might belong in a separate issue - please let me know is one should be created.

OM already shows "Wikimedia Commons" link if the selected object has the wikimedia_commons tag.
It would be great if it was possible to show image for "File:" objects inline in OM, without opening it in the browser - similar to how it is done for Wikipedia articles.
For network traffic considerations, this could potentially be triggered by a "download image" action or similar.

**Comment by biodranik** [comment]
Aren't there already some issues about displaying images from wikimedia and image tags?

**Comment by richlv** [comment]
I searched for something specifically about showing wikimedia images, could only find this issue.
